pulp_rpm/app/tasks/utils.py

```diff
@@ -0,0 +1,186 @@
+
+
+class KickstartData:
```

Comment:
Is there a tool available that actually parses the treeinfo files for us? I'm not 100% sure, but I think this python library provides some things  https://release-engineering.github.io/productmd/index.html   Is there something useful in there that could fill the need this object provides?

Reply:
I have to do it for modeling because of some differences,
 `Image` for example:
```
[images-x86_64]
boot.iso = images/boot.iso
initrd = images/pxeboot/initrd.img
kernel = images/pxeboot/vmlinuz
upgrade = images/pxeboot/upgrade.img
```
the `productmd` returns:
```python
{"images-x86_64": {"boot.iso": "images/boot.iso", ...}}
```
So I needed this object to turn it into:
```python
[{"image": {"platform":x86_64, "name":"boot.iso", "path": "images/boot.iso"}}, ...]
```

Reply:
I see. This makes sense. :+1: